core/src/main/java/org/elasticsearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -609,16 +616,19 @@ public boolean escape() {
     }
 
     /**
-     * Whether query text should be split on whitespace prior to analysis.
-     * Default is <code>{@value #DEFAULT_SPLIT_ON_WHITESPACE}</code>.
+     * This query parser splits on operator only.
```

Comment:
Maybe explicitly state that this setting is ignored like `autoGeneratePhraseQueries`?